briebug/ngrx-entity-schematic
# Feat/schematic

- Restructure template app into sibling folder with schematic
- add script for copying and modifying template app files into schematic dir

## To test

1. pull down, run `yarn` in root dir and `/schematic-src`
1. cd into `/schematic-src.
1. run `ng test` and `ng-s` and make sure things still work
